Formula/git-lfs.rb

```diff
@@ -18,6 +18,17 @@ def install
     bin.install "bin/git-lfs"
   end
 
+  def caveats; <<-EOS.undent
+    Update your git config to finish installation:
+
+      # Update global git config (~/.gitconfig)
+      $ git lfs install
+
+      # Update system git config (/etc/gitconfig)
```

Comment:
Potentially not worth listing the location for the system git config, because depending on which `git` you use & where it came from, `/etc/gitconfig` may not be accurate.